roles/lifecycle_environments/README.md

```diff
@@ -0,0 +1,68 @@
+theforeman.foreman.lifecycle_environments
+=========================================
+
+This role creates and manages Lifecycle Environments.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `lifecycle_environments`. Each `lifecycle_environment` requires the following fields:
+
+- `name`: The name of the lifecycle environment.
+- `prior`: The name of the previous lifecycle environment to attach to in sequence. For the first lifecycle environment in a new path, set the prior lifecycle environment to Library.
+
+The following fields are optional and will be omitted by default:
+
+- `description`: Description of the lifecycle environment
+- `label`: A permanent label for identifying the lifecycle environment to tools such as subscription-manager. This is created by the server if omitted.
+
+Example Playbooks
+-----------------
+
+Create a lifecycle environment path with three environments:
+
+```yaml
+- hosts: localhost
+  roles:
+    - role: theforeman.foreman.lifecycle_environments
+      vars:
+        server_url: https://foreman.example.com
+        username: "admin"
+        password: "changeme"
+        organization: "Default Organization"
+        lifecycle_environment_paths:
+          - name: "Dev"
+            prior: "Library"
+          - name: "Test"
+            prior: "Dev"
+          - name: "Prod"
+            prior: "Dev"
+```
+
+Create two lifecycle environment paths:
```

Comment:
Should we call out the paths here in the description, so the users know them before looking at the YAML and building paths in their head?